src/games/strategy/util/PointFileReaderWriter.java

```diff
@@ -149,6 +149,31 @@ public static void writeOneToMany(final OutputStream sink, Map mapping) throws E
     }
     write(out, sink);
   }
+  
+  /*public static void writeOneToMany(final OutputStream sink, Map<String, List<Point>> mapping) throws Exception {
```

Comment:
No worries, context is a bit immaterial here. IMO the best way to respond to PR review comments is by updating the code, let the code do the talking for itself. When you hit that point, it's a good sign that the code is maintainable. After this PR is closed, all the commentary in it will be rendered irrelevant. So placating me in this specific review is not the goal, the goal is to have it so that anyone reading this code will not have the same questions / comments.